src/commands/dataconnect-sql-migrate.ts

```diff
@@ -34,6 +34,7 @@ export const command = new Command("dataconnect:sql:migrate [serviceId]")
       options,
       schema: serviceInfo.schema,
       validateOnly: true,
+      schemaValidation: serviceInfo.dataConnectYaml.schema.datasource.postgresql?.schemaValidation,
```

Comment:
Quick sanity check - does this get broken by our move from datasource -> datasources? @fredzqm might know off the top of his head.

Reply:
Chatted offline -- based on go/fdc:multi-data-source it looks like the `datasource` field will still be supported, though we should also support `datasources` in #7589.